src/main/java/dev/openfga/sdk/api/configuration/Configuration.java

```diff
@@ -259,4 +270,13 @@ public Configuration minimumRetryDelay(Duration minimumRetryDelay) {
     public Duration getMinimumRetryDelay() {
         return minimumRetryDelay;
     }
+
+    public Configuration additionalHeaders(Map<String, String> additionalHeaders) {
```

Comment:
In the configuration, can we call it `defaultHeaders` to be consistent?

- https://github.com/openfga/go-sdk/blob/main/configuration.go#L45C2-L45C16
- https://github.com/openfga/dotnet-sdk/blob/main/src/OpenFga.Sdk/Configuration/Configuration.cs#L99

Reply:
Updated in https://github.com/openfga/java-sdk/pull/37/commits/23603940828703435405356892b57c14add3cfcb